spec/ioki/model/models_without_open_api_specifications_spec.rb

```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+RSpec.describe 'models without open api specifications', vcr: { record: :all } do
```

Comment:
I split this file up and their shouldn't be `record: :all` on CI. Let me have a look in master.

Reply:
Yes. Could you start from/rebase on master?

Reply:
Actually deleting the file and the vcr cassettes should do the trick.

Reply:
Yes. I don't know why the WIP commits are in this PR at all.